store/src/main/java/com/dropbox/android/external/store4/SourceOfTruth.kt

```diff
@@ -105,22 +105,22 @@ interface SourceOfTruth<Key, Input, Output> {
         )
 
         /**
-         * Creates a ([Flow]) source of truth that is accessed via [reader], [writer], [delete] and
-         * [deleteAll].
+         * Creates a ([Flow]) source of truth that is accessed via [flowReader], [writer],
+         * [delete] and [deleteAll].
          *
-         * @param reader function for reading records from the source of truth
+         * @param flowReader function for reading records from the source of truth
          * @param writer function for writing updates to the backing source of truth
          * @param delete function for deleting records in the source of truth for the given key
          * @param deleteAll function for deleting all records in the source of truth
-         *
          */
-        fun <Key : Any, Input : Any, Output : Any> from(
-            reader: (Key) -> Flow<Output?>,
+        @JvmName("ofFlow")
+        fun <Key : Any, Input : Any, Output : Any> of(
+            flowReader: (Key) -> Flow<Output?>,
```

Comment:
assuming you land on "this is discoverable" should we call the flow one `reader` and the non flow one `nonFlowReader` to highlight the better default?

Reply:
updated. i hate it for anyone needs to call it but i think it is the right call.